pkg/scannerv4/mappers/mappers.go

```diff
@@ -1218,6 +1219,28 @@ func FindName(vuln *claircore.Vulnerability, p *regexp.Regexp) (string, bool) {
 	return "", false
 }
 
+// advisory returns the vulnerability's related advisory.
+//
+// Only Red Hat advisories (RHSA/RHBA/RHEA) are supported at this time.
+func advisory(vuln *claircore.Vulnerability) string {
```

Comment:
@crozzy can you confirm we can/should fetch the advisory from the links?

Reply:
There's no API guarantee for the format/contents of the links field, having said that:
* I don't think there is any other way to tie them together without pulling the VEX CVE file and looking it up yourself.
* There is no plans to change the format until the Matcher v2 schema changes are updated that should render hacks like this moot.

Reply:
works for me. Seems like this is the best way to do this using Claircore's native data